### PR TITLE
Donator item

### PR DESCRIPTION
**Pull request by tf-4**
image

### CODE REVIEW

File: modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm [hunk 1]
```diff
@@ -502,3 +502,8 @@
 	item_path = /obj/item/clothing/under/bubbly_clown/skirt
 	restricted_roles = list(JOB_CLOWN)
 	ckeywhitelist = list("boisterousbeebz", "aether217")
+
+/datum/loadout_item/pocket_item/marsoc_coin
+	name = "MARSOC Challenge Coin"
+	item_path = /obj/item/coin/donator/marsoc
+	ckeywhitelist = list("Sweetsoulbrother")
```
Comment (tf-4) [inline modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm]:
```suggestion
	ckeywhitelist = list("sweetsoulbrother")
```

Comment (Zonespace27) [inline modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm]: ckey santization

**Review by Useroth — APPROVED** [review]

**Comment by mazzinsanity** [comment]
There's strong, and then there's Army Strong. 😎🎖🪖🇺🇸🇺🇸🇺🇸

**Comment by electrical-pants** [comment]
allowing a piece of memorabilia from a US military branch into a scifi rp server 💀

**Comment by Zonespace27** [comment]
Useroth approved

**Comment by sweetsoulbro** [comment]
> allowing a piece of memorabilia from a US military branch into a scifi rp server 💀

SolFed Military Branch. Solfed Marine Corps. The artist just used "USMC" cause of the link I gave them.

**Pull request merged**